pages/1.10/security/ent/tls-ssl/ca-custom/index.md

```diff
@@ -8,26 +8,310 @@ excerpt:
+- Custom CA certificates are only supported for a fresh installation of DC/OS Enterprise 1.10 or higher. Older versions of DC/OS are not supported, and it is not possible to add a custom CA certificate during an upgrade.
 
-**Note:** 
+# Glossary of the terms used in this documentation
+- **Custom CA certificate:** Your custom CA certificate in the PEM format, which will be used to issue certificates for DC/OS components such as Admin Router. The custom CA certificate is either an intermediate CA certificate (issued by another CA) or a root CA certificate (self-signed by the custom CA).
 
-- Only custom CA certificates that have an associated RSA-type key pair are supported.
-- You can use a custom CA certificate only with a new 1.10 install.
+- **Private key associated with the custom CA certificate:** The private key in the PKCS#8 format associated with the custom CA certificate.
 
-# Procedure
+- **Certificate chain associated with the custom CA certificate:** The complete CA certification chain required for end-entity certificate verification. It must include the certificates of all parent CAs of the intermediate custom CA up to and including the root CA certificate. If the custom CA certificate is a root CA certificate, the chain must be empty.
 
-1. Using a [custom setup file and the CLI or advanced installer](/1.10/installing), provide the paths to the custom CA certificate, corresponding private key, and optionally corresponding intermediate CA certificates and/or a root CA certificate as part of the chain. All files must be placed in `genconf` directory in the home directory of your bootstrap node. The installer verifies the data.
-1. Before installing DC/OS to all nodes, place the private key file corresponding to the custom CA certificate specified in `ca_certificate_key_path` securely on all master nodes in `/var/lib/dcos/pki/tls/CA/private/custom_ca.key`. The file must be readable by the root user and have 0600 permissions set.
+- **Installation directory:** The directory on the bootstrap node where the DC/OS installer resides. It is denoted with `$DCOS_INSTALL_DIR` in this document.
+
+- **Configuration:** The set of the configuration parameters that governs the specific aspects of the installation procedure. The configuration is stored in the DC/OS configuration file.
+
+- **DC/OS configuration file:** The file which contains the DC/OS configuration parameters. The DC/OS configuration file is normally called `config.yaml` and must be present in the `$DCOS_INSTALL_DIR/genconf/` directory on the bootstrap node during the installation. It is used by the DC/OS installer.
+
+
+# Requirements
+
+In order to install DC/OS Enterprise with a custom CA certificate you will need:
+
+- to use the [advanced DC/OS installation method](/1.10/installing/ent/custom/advanced/). Other installation methods are not supported.
```

Comment:
For reviewer from the doc team:
Please check that the link points correctly to the following page:
https://docs.mesosphere.com/1.10/installing/ent/custom/advanced/
Thank you!

Reply:
The link seems to be correct.